tensorrt_llm/_torch/modules/fused_moe/communication/communication_factory.py

```
@@ -140,8 +140,8 @@ def create_strategy(
             )
             logger.info("Selected communication strategy: NVLinkOneSided")
             return strategy
-        except RuntimeError as e:
-            logger.debug(f"NVLinkOneSided not available: {e}")
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)